.scripts/create_push_client.sh

```diff
@@ -17,54 +17,58 @@ function generate_client_and_create_pr() {
+    done
+    rm -rf /tmp/${GHREPO}/cluster /tmp/${GHREPO}/tool
     cp -r cluster tool /tmp/${GHREPO}
-    git rev-parse --short HEAD > /tmp/${GHREPO}/source_commit.txt
+    git rev-parse HEAD > /tmp/${GHREPO}/source_commit.txt
     cd /tmp/${GHREPO}
 
     git commit cluster tool source_commit.txt -m "${message}"
     git push -u origin ${branch}
-    rm -rf /tmp/${GHREPO}
+    cd - & rm -rf /tmp/${GHREPO}
 
     set +x
-    curl -s -X POST -L -H "Authorization: token $(echo ${FABRIC8_HUB_TOKEN}|base64 --decode)" \
-         -d "{\"title\": \"${message}\", \"body\": \"${body}\", \"base\":\"master\", \"head\":\"${branch}\"}" \
+    curl -i -s -X POST -L -H "Authorization: token $(echo ${FABRIC8_HUB_TOKEN}|base64 --decode)" \
+         -d "{\"title\": \"${message}\", \"body\": \"$(echo $body)\", \"base\":\"master\", \"head\":\"${branch}\"}" \
          https://api.github.com/repos/${GHORG}/${GHREPO}/pulls
     set -x
 }
 
 function pr_body() {
-    local body=$(cat <<EOF
-    # About
-    This description was generated using following command:
-    \`\`\`sh
+    local description=$(cat <<EOF
+            **About**<br><br>
+            This description was generated using following command:<br><br>
+            \`\`\`
 
-    `echo GHORG=${GHORG} GHREPO=${GHREPO} LAST_USED_COMMIT=${LAST_USED_COMMIT} LATEST_COMMIT=${LATEST_COMMIT} \
-    git log --pretty="%n**Commit:** https://github.com/${GHORG}/${GHREPO}/commit/%H%n**Author:** %an (%ae)%n**Date:** %aI%n%n%s%n%n%b%n%n----%n" \
-            --reverse ${LAST_USED_COMMIT}..${LATEST_COMMIT} \
-            | sed -E "s/([\s|\(| ])#([0-9]+)/\1${GHORG}\/${GHREPO}#\2/g"`
+            `echo GHORG=${GHORG} GHREPO=${GHREPO} LAST_USED_COMMIT=${LAST_USED_COMMIT} LATEST_COMMIT=${LATEST_COMMIT} \
+             git log --pretty="%n**Commit:** https://github.com/${GHORG}/${GHREPO}/commit/%H%n**Author:** %an (%ae)%n**Date:** %aI%n%n" --reverse ${LAST_USED_COMMIT}..${LATEST_COMMIT} design
+           `
 
-    \`\`\`
-
-    # Changes
+            \`\`\`
+            <br><br>
+            **Commits with change in Design Package**<br><br>
 EOF
-    git log \
-      --pretty="%n**Commit:** https://github.com/${GHORG}/${GHREPO}/commit/%H%n**Author:** %an (%ae)%n**Date:** %aI%n%n%s%n%n%b%n%n----%n" \
-      --reverse ${LAST_USED_COMMIT}..${LATEST_COMMIT} \
-      | sed -E "s/([\s|\(| ])#([0-9]+)/\1${GHORG}\/${GHREPO}#\2/g"
 )
 
-    echo $body
+    local commits=$(git log --pretty="**Commit:** https://github.com/${GHORG}/${GHREPO}/commit/%H<br>**Author:** %an (%ae)<br>**Date:** %aI<br><br>" --reverse ${LAST_USED_COMMIT}..${LATEST_COMMIT} design)
+
+    echo $description$commits
 }
 
 function generate_client_setup() {
     SERVICE_NAME=${PWD##*/}
-    GHORG=${1:-fabric8-services}
-    GHREPO=${2:-${SERVICE_NAME}-client}
+    PKG_NAME=${1}
+    TOOL_DIR=${2:-tool/cli}
+    GHORG=${3:-fabric8-services}
```

Comment:
I think we need to add some comments here with descriptions of the params we expect here. And how they will be used.
GitHub Org and Repo are self-explanatory but new params are not.